papers/Mathematics/paper.Rmd

```diff
@@ -0,0 +1,269 @@
+  In this paper, we provide an overview of current recommandations and best
+  practices and demonstrate how they can easily and conveniently be 
+  implemented in the R statistical computing software, using the *{performance}*
+  package of the *easystats* ecosystem. We cover univariate, multivariate, and 
+  model-based statistical outlier detection methods, their recommended 
+  threshold, standard output, and plotting methods. We conclude with 
+  recommendations on the handling of outliers: the different theoretical types 
+  of outliers, whether to exclude or winsorize them, and the importance of 
+  transparency.
+keywords: |
+  univariate outliers; multivariate outliers; robust detection methods; R; easystats
+acknowledgement: |
+  *{performance}* is part of the collaborative 
+  [*easystats*](https://github.com/easystats/easystats) ecosystem 
+  [@easystatspackage]. Thus, we thank all 
+  [members of easystats](https://github.com/orgs/easystats/people), 
+  contributors, and users alike.
+authorcontributions: |
+  R.T. drafted the paper; all authors contributed to both the writing of the 
+  paper and the conception of the software.
+funding: |
+  This research received no external funding.
+conflictsofinterest: |
+  The authors declare no conflict of interest.
+abbreviations:
+  - short: SOD
+    long: Statistical outlier detection
+  - short: SEM
+    long: Structural equation modelling
+  - short: SD
+    long: Standard deviation
+  - short: MAD
+    long: Median absolute deviation
+  - short: IQR
+    long: Interquartile range
+  - short: HDI
+    long: Highest density interval
+  - short: BCI
+    long: Bias corrected and accelerated interval
+  - short: MCD
+    long: Minimum covariance determinant
+  - short: ICS
+    long: invariant coordinate selection
+  - short: OSF
+    long: Open Science Framework
+output: rticles::mdpi_article
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  comment = "#>",
+  out.width = "100%",
+  dpi = 300,
+  warning = FALSE
+)
+
+library(performance)
+library(see)
+library(datawizard)
+```
+
+# Introduction
+
+Real-life data often contain observations that can be considered *abnormal* when compared to the main population. The cause of it---be it because they belong to a different distribution (originating from a different generative process) or simply being extreme cases, statistically rare but not impossible---can be hard to assess, and the boundaries of "abnormal" are hard to define.
+
+Nonetheless, the improper handling of these outliers can substantially affect statistical model estimations, biasing effect estimations and weakening the models' predictive performance.
+It is thus essential to address this problem in a thoughtful manner. Despite the existence of established recommendations and guidelines, many researchers still do not treat outliers in a consistent manner, or do so using inappropriate strategies [@simmons2011false; @leys2013outliers].
```

Comment:
I included it because of this:

> In a recent article, Simmons, Nelson, and Simonsohn (2011) showed how, due to the misuse of statistical tools, significant results could easily turn out to be false positives (i.e., effects considered significant whereas the null hypothesis is actually true). In their argument, they accurately pinpointed the importance of outliers.

From the Simmons paper: 

> Ambiguity is rampant in empirical research. As an example, consider a very simple decision faced by researchers analyzing reaction times: how to treat outliers. In a perusal of roughly 30 Psychological Science articles, we discovered considerable inconsistency in, and hence considerable ambiguity about, this decision.